src/libmongoc/src/mongoc/mongoc-error.c

```diff
@@ -20,6 +20,7 @@
 #include "mongoc-error-private.h"
 #include "mongoc-rpc-private.h"
 #include "mongoc-client-private.h"
+#include "mongoc-server-description-private.h"
 
 bool
 mongoc_error_has_label (const bson_t *reply, const char *label)
```

Comment:
Aside: are error labels expected in `writeConcernError`?

DRIVERS-1641 suggests `writeConcernError` may have error labels:
> Should drivers consider the code/label within the writeConcernError field from a mongos reponse?
> - YES for labels from a 4.4+ mongos. The spec already says that mongos will avoid adding the label if it already retried to help avoid "excessive retries" by the driver.

DRIVERS-1385 suggests `writeConcernError` may not have error labels:
> The server always returns error labels as top-level fields

`mongoc_error_has_label` checks `writeConcernError` for labels. That may not be necessary. Though I do not think it needs to be addressed in this PR.

Reply:
> Aside: are error labels expected in writeConcernError?

No. [DRIVERS-1385](https://jira.mongodb.org/browse/DRIVERS-1385) is correct, since that was confirmed in [this comment](https://jira.mongodb.org/browse/DRIVERS-1385?focusedCommentId=3406472&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3406472). I'll revise the description of [DRIVERS-1641](https://jira.mongodb.org/browse/DRIVERS-1641) accordingly.

I had no intention of adding label-checking behavior for the `writeConcernError` document, but also missed that libmongoc was already doing so. Thanks for catching that.

I created [CDRIVER-4808](https://jira.mongodb.org/browse/CDRIVER-4808) to track the `mongoc_error_has_label` cleanup.